keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+#include "ergodox.h"
+// #include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+#include "eeconfig.h"
+#include "eeprom.h"
+#include "keymap_colemak.h"
+
+extern keymap_config_t keymap_config;
+
+enum ergodox_layers {
+  _QWERTY,
+  _COLEMAK,
+  _QWOC,
+  _LOWER,
+  _RAISE,
+  _PLOVER,
+// Intermediate layers for SuperDuper (Combo keys does not work on Infinity yet)
+  _SUPER,
+  _DUPER,
+  _SUPERDUPER,
+  _MOUSE,
+  _ADJUST,
+  _MDIA,
+  _SYMB,
+};
+
+enum ergodox_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  QWOC,
+  PLOVER,
+  SUPER,
+  DUPER,
+  SUPERDUPER,
+  MOUSE,
+  LOWER,
+  RAISE,
+  BACKLIT,
+  EXT_PLV,
+  SDTOGG, // Toggle SuperDuper
+  EPRM,
+  VRSN,
+  RGB_SLD
+};
+
+enum functions {
+  M_GUI_UNDS, // Simulate GUI_T(KC_UNDS)
+  M_SFT_PO, // SFT_T(KC_LPRN)
+  M_SFT_PC, // SFT_T(KC_RPRN)
+};
+
+// Timer for custom mod tap
+static uint16_t m_gui_unds_timer;
+static uint16_t m_sft_po_timer;
+static uint16_t m_sft_pc_timer;
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```